airflow/www/static/js/App.tsx

```diff
@@ -61,16 +51,24 @@ const queryClient = new QueryClient({
   },
 });
 
+interface ChildrenProps {
+  children: React.ReactNode;
+}
+
+interface AppProps extends ChildrenProps {
+  cache: EmotionCache;
+}
+
 // Chakra needs to access the containerRef provider so our tooltips pick up the correct styles
-const ChakraApp = () => {
+const ChakraApp = ({ children }: ChildrenProps) => {
```

Comment:
You can import `PropsWithChildren` from React to simplify a bit:

```tsx
import React, { PropsWithChildren } from 'react';
```

```suggestion
const ChakraApp = ({ children }: PropsWithChildren) => {
```